platform/android/export/export_plugin.cpp

```diff
@@ -29,6 +29,7 @@
 /*************************************************************************/
 
 #include "export_plugin.h"
+#include "godot/editor/export_template_manager.h"
```

Comment:
I had to change this include to `editor/export_template_manager.h` to be able to build